### PR TITLE
JW-928 agent as a service

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Review by jablonskim — APPROVED** [review]

File: src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<service>
+    <id>ContrailAgent</id>
+    <name>ContrailAgent</name>
+    <description>Contrail vrouter agent</description>
+    <executable>contrail-vrouter-agent.exe</executable>
+    <arguments>--config_file=contrail-vrouter-agent.conf</arguments>
```
Comment (mclapinski) [inline src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml]: This is useless

File: src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<service>
+    <id>ContrailAgent</id>
+    <name>ContrailAgent</name>
+    <description>Contrail vrouter agent</description>
```
Comment (mclapinski) [inline src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml]: I'd suggest `Contrail vRouter Agent`

**Review by mclapinski — CHANGES_REQUESTED** [review]

File: src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<service>
+    <id>ContrailAgent</id>
+    <name>ContrailAgent</name>
+    <description>Contrail vRouter Agent</description>
+    <executable>contrail-vrouter-agent.exe</executable>
+    <logmode>none</logmode>
```
Comment (mclapinski) [inline src/vnsw/agent/contrail/windows/service_config/service-wrapper.xml]: please change to `append` or something

**Review by mclapinski — CHANGES_REQUESTED** [review]

**Review by mclapinski — APPROVED** [review]

**Comment by PawelMarchewkaCodiLime** [comment]
Janie, przetestuj, proszę.

**Comment by PawelMarchewkaCodiLime** [comment]
Retest this please

**Pull request merged**